testing/regress/ecl/memcachedtest.ecl

```diff
@@ -17,6 +17,7 @@
 
 //class=embedded
 //class=3rdparty
+//nothor
```

Comment:
I assume this is added because the new test doesn't work in thor.  Was there a problem running the rest of the tests in thor?  If not then it would be better to avoid just that test in thor.  (E.g., by testing \_\_PLATFORM\_\_)